libpod/image/image.go

```diff
@@ -74,6 +74,11 @@ type InfoImage struct {
 	Layers []LayerInfo
 }
 
+// Filter is a function to determine whether a image is included
+// in command output. Images to be outputted are tested using the function.
+// A true return will include the image, a false return will exclude it.
+type Filter func(*Image) bool
```

Comment:
Can we rename this to `ImageFilter`? I can see us adding Container filters and Pod filters in the future, too, which might give us name collisions.

Reply:
I initially named this `Filter` as `ImageFilter`. But `make validate` reports error as its part of image package and user need to user `image.ImageFilter`.

Reply:
Can we either tell validate to be quiet or make it plural
ImagesFilter

Reply:
Done. used `//nolint` for ignoring the check against `ImageFilter`